### PR TITLE
164 new typography

### PR DESCRIPTION
**Pull request by byjs-dev**
This branch includes the typo styles from the latest ZDS.sketch - Abstact Design System Master branch (https://share.goabstract.com/c5361928-c08e-44b0-8aa5-88f728017fec?mode=design&sha=b3b296594b96f9ec49124881611731e3b18a90c3).
Text colors are not part of typo stylesheet and should be added by a theme or individually.
Text alignment isn't part of the typo stylesheet, as well. A newly included "helpers" stylesheet will provide extra classes for that.

### CODE REVIEW

File: src/style-basics/_helpers.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@mixin helpers {
+  .#{$prefix}font-weight--bold {
+    font-weight: 700 !important;
```
Comment (byjs-dev) [inline src/style-basics/_helpers.scss:3]: Yeah, we'll find a cleaner solution soon. ;)

File: src/themes/_color_themes.scss [hunk 2]
```diff
@@ -80,3 +80,7 @@ $color_themes: (
 @function themed($key) {
   @return map-get($theme-map, $key);
 }
+
+@function themed2($keys...) {
+  @return map-deep-get($theme-map, $keys...);
+}
```
Comment (byjs-dev) [inline src/themes/_color_themes.scss:86]: Ah, this is a leftover from what I usually want to work on... implementing the font color shades...
It's ok to leave it here, it's currently not in use. I'll work on that later in another feature branch.

**Review by cabutler10 — APPROVED** [review]

**Comment by byjs-dev** [comment]
The last update was basically a refactoring to eliminate duplicate code. Typo classes are now generated from the typo data of guidelines.json (-> guidelines.scss). From now on all values should only be edited there.

**Review by cabutler10 — APPROVED** [review]

**Review by cabutler10 — APPROVED** [review]

**Comment by byjs-dev** [comment]
I had to resolve conflicts by merging develop into this branch. The new icons from develop were moved into another folder to prevent them from being overwritten by the build process. The usage of new icons will be done in another branch.

**Pull request merged**